test/cql-pytest/test_tools.py

```diff
@@ -1232,3 +1233,19 @@ def test_scylla_sstable_bad_scylla_yaml(cql, test_keyspace, scylla_path, scylla_
                     "error processing configuration item: Unknown option : foo",
                     "Successfully read scylla.yaml from"):
                 assert any(map(lambda stderr_line: expected_msg in stderr_line, stderr_lines))
+
+
+def test_scylla_sstable_format_version(cql, test_keyspace, scylla_data_dir):
```

Comment:
I'm not sure this package is the best place for this test.
@denesb would probably won't allow it as it's very specific to scylla tools :)